packages/block-editor/src/components/block-patterns-list/style.scss

```diff
@@ -71,3 +71,10 @@
 		}
 	}
 }
+// Solution for "Add "selected" styling to active elements in the design selector."
+// https: //github.com/WordPress/gutenberg/issues/65127
+.block-editor-block-patterns-list__list-item.is-active {
+	border: 2px solid #007cba;
+	/* Custom styles for active item */
+	background-color: #eaf5ff;
+}
```

Comment:
Let's make sure we uniform to how we apply focused/selected styles in other parts of the editor. Just looking [a few lines above](https://github.com/WordPress/gutenberg/blob/cd77dd950b7762ebfa99987ffb0b3ed8f9888d94/packages/block-editor/src/components/block-patterns-list/style.scss#L53-L55), I see that we're already using `outline` (and not border), and that we're consuming a bunch of CSS variables for the color and thickness.

Reply:
Thanks for mentioning this point @ciampo 

I will update this css in the next commit.

Reply:
CSS has been updated as mentioned.